src/test/regress/expected/metadata_sync_helpers.out

```diff
@@ -683,6 +757,72 @@ BEGIN TRANSACTION ISOLATION LEVEL READ COMMITTED;
 
 (6 rows)
 
+COMMIT;
+-- try to update placements
+-- fails because we are trying to update it to non-existing node
+BEGIN TRANSACTION ISOLATION LEVEL READ COMMITTED;
+	SELECT assign_distributed_transaction_id(0, 8, '2021-07-09 15:41:55.542377+02');
+ assign_distributed_transaction_id
+---------------------------------------------------------------------
+
+(1 row)
+
+	SET application_name to 'citus';
+	\set VERBOSITY terse
+	SELECT citus_internal_update_placement_metadata(1420000, get_node_id(), get_node_id()+1000);
+ERROR:  Node with group id 1014 for shard placement xxxxx does not exist
+COMMIT;
+-- fails because the source node doesn't contain the shard
+BEGIN TRANSACTION ISOLATION LEVEL READ COMMITTED;
+	SELECT assign_distributed_transaction_id(0, 8, '2021-07-09 15:41:55.542377+02');
+ assign_distributed_transaction_id
+---------------------------------------------------------------------
+
+(1 row)
+
+	SET application_name to 'citus';
+	\set VERBOSITY terse
+	SELECT citus_internal_update_placement_metadata(1420000, get_node_id()+10000, get_node_id());
+ERROR:  Active placement for shard xxxxx is not found on group:14
+COMMIT;
+-- fails because invalid shardid
+BEGIN TRANSACTION ISOLATION LEVEL READ COMMITTED;
+	SELECT assign_distributed_transaction_id(0, 8, '2021-07-09 15:41:55.542377+02');
+ assign_distributed_transaction_id
+---------------------------------------------------------------------
+
+(1 row)
+
+	SET application_name to 'citus';
+	\set VERBOSITY terse
+	SELECT citus_internal_update_placement_metadata(0, get_node_id(), get_node_id()+1);
+ERROR:  Shard id does not exists: 0
```

Comment:
Error doesn't match description of test

Reply:
I initially had `if shardid <= invalidShardID` check, but then I realized that `shardExists()` already covers that so removed. I'll update the comment